ingest/scripts/call_loculus.py

```diff
@@ -327,6 +328,10 @@ def get_submitted(config: Config):
         logger.error(f"Error decoding JSON from /get-original-metadata: {response_summary}")
         raise ValueError from err
 
+    if len(entries) != expected_record_count:
```

Comment:
Only wonder if we should retry here instead of raising an error. The case `len(entries) != expected_record_count` is most likely to happen at the start of ingest and we then have to wait 30min until the approve rule finishes for the pods to restart.

Reply:
How about this now?

Reply:
https://github.com/loculus-project/loculus/pull/2857/commits/b75314da6ef0f90d5ee6cea0339792a3bbc05366

Reply:
Looks good! Lets wait till ingest succeeds again incase I overlooked sth in the code then feel free to merge :-)

Reply:
Looks good, but let's just wait a bit more because why not :)

![Uploading image.png…]()

Reply:
Reingest went well, logs look good - marking this as resolved